src/lib/components/Codelab/index.js

```diff
@@ -85,6 +88,14 @@ class Codelab extends BaseElement {
       return html``;
     }
     const isTest = env === 'test';
```

Comment:
In what situation this is true? Do we have a test for this component? Is there another scenatio where env === test that I just don't recall?